debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/DomainTypesIT.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.connector.postgresql;
+
+import java.sql.SQLException;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import io.debezium.connector.postgresql.PostgresConnectorConfig.SnapshotMode;
+import io.debezium.doc.FixFor;
+import io.debezium.junit.SkipWhenDatabaseVersion;
+
+import static io.debezium.junit.EqualityCheck.LESS_THAN;
+
+/**
+ * Integration test to verify behaviour of tables which include domain types
+ *
+ * @author Richard Kolkovich <richard@sigil.org>
+ */
+@SkipWhenDatabaseVersion(check = LESS_THAN, major = 11, minor = 0, reason = "Database Version less than 11")
```

Comment:
Let's make it a bit more descriptive:

```suggestion
@SkipWhenDatabaseVersion(check = LESS_THAN, major = 11, minor = 0, reason = "Domain type syntax not supported")
```